.github/workflows/update_astropy_iers_data_pin.yml

```diff
@@ -0,0 +1,54 @@
+# Regularly update the minimum version of astropy-iers-data in setup.cfg,
+# to ensure that if users update astropy, astropy-iers-data will also get
+# updated to a recent version.
+
+name: Auto-update astropy-iers-data minimum version
+
+on:
+  schedule:
+    - cron: '0 0 1 * *'  # Monthly
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+
+  update-astropy-iers-data-pin:
+    permissions:
+      contents: write  # for peter-evans/create-pull-request to create branch
+      pull-requests: write  # for peter-evans/create-pull-request to create a PR
+    name: Auto-update astropy-iers-data minimum version
+    runs-on: ubuntu-latest
+    if: github.repository == 'astropy/astropy'
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.x
+    - name: Install dependencies
+      run: pip install requests
+    - name: Run update script
+      run: python .github/workflows/update_astropy_iers_data_pin.py
+    - name: Commit changes
+      run: |
+        git config user.name github-actions
+        git config user.email github-actions@github.com
+        git add setup.cfg
+        if ! git diff --cached --exit-code; then
+          git commit -m "Update minimum required version of astropy-iers-data"
+        fi
+    - name: Create Pull Request
+      uses: peter-evans/create-pull-request@v5
+      with:
```

Comment:
I see above that you only make a commit if there is an actual change. But if there is no change, will there still be a PR created?

Reply:
If there are no commits different compared to the PR base, no PR is opened